index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+  <header id="header">
+    <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/0/09/Snickers_logo.svg/1200px-Snickers_logo.svg.png"
+      alt="" id="header-img">
+    <nav id="nav-bar">
+      <a href="#section1" class="nav-link">Section 1</a>
+      <a href="#section2" class="nav-link">Section 2</a>
+      <a href="#section3" class="nav-link">Section 3</a>
+    </nav>
+  </header>
+  <section id="section1">
+    <p>Snickers (stylized as SNICKERS) is a brand name chocolate bar made by the American company Mars,
+      Incorporated, consisting of nougat topped with caramel and peanuts that has been enrobed in milk chocolate.</p>
+    <img src="http://assets.stickpng.com/images/580b57fbd9996e24bc43c0c4.png" alt="" id="img2">
+  </section>
+  <section id="section2">
```

Comment:
i think it would be better if you use the semantic 'main' tag. why don't you use the tag instead of repeat the section tag since it would give better positioning in the web browser and give better HTML structure to the page. 🦾

Reply:
Nice job @elassis 👍